wof/examples/flask/odm2/timeseries/odm2_config_timeseries.cfg

```diff
@@ -16,8 +17,8 @@ EndDate: 2007-09-01T02:30:00
 
 [WOF_1_1]
 # Change this to your configuration; assumes port 8080
-# http://serverip:port/networkcode/soap/cuahsi_1_1/.wsdl 
-Service_WSDL: http://127.0.0.1:8080/odm2timeseries/soap/cuahsi_1_1/.wsdl 
+# http://serverip:port/urlpath/soap/cuahsi_1_1/.wsdl
+Service_WSDL: http://127.0.0.1:8080/wofpysqlite/soap/cuahsi_1_1/.wsdl
```

Comment:
See my other comment on `test_config.cfg` about `Service_WSDL` now being obsolete (and even worse, if it's inconsistent with `URLPATH`).